queries/clojure/highlights.scm

```diff
@@ -103,7 +103,7 @@
 ((sym_lit) @method
  (#lua-match? @method "^[.][^-]"))
 ((sym_lit) @field
- (#lua-match? @field "^[.]-"))
+ (#lua-match? @field "^\\.-"))
```

Comment:
What about %. instead of \\., since it's lua-match

Reply:
Sure, I'll try that out.